ScottWu.md

```diff
@@ -1,24 +1,10 @@
-## 09/07 ~ 09/11
+## 09/14 ~ 09/18
 
 - [Settings]
-	- [Fixed] [Bug 1179343](https://bugzilla.mozilla.org/show_bug.cgi?id=1179343) - WiFi hotspot settings
-	- [Fixed] [Bug 1199872](https://bugzilla.mozilla.org/show_bug.cgi?id=1199872) - Turning sound off makes a sound
+	- [Reviewing] [Bug 1014442] (https://bugzilla.mozilla.org/show_bug.cgi?id=1014442) - refactor Find My Device panel with AMD pattern
```

Comment:
I'll be working on higher priority bugs this week. 

@gasolin and @evanxd have already passed me a few things I can work on.